go-controller/pkg/ovn/egressip.go

```diff
@@ -399,65 +425,65 @@ func (oc *Controller) getSortedEgressData() ([]egressNode, map[string]bool) {
 	return assignableNodes, allAllocations
 }
 
-func (oc *Controller) setNodeEgressAssignable(nodeName string, isAssignable bool) {
-	oc.eIPC.allocatorMutex.Lock()
-	defer oc.eIPC.allocatorMutex.Unlock()
-	if eNode, exists := oc.eIPC.allocator[nodeName]; exists {
+func (e *egressIPController) setNodeEgressAssignable(nodeName string, isAssignable bool) {
+	e.allocatorMutex.Lock()
+	defer e.allocatorMutex.Unlock()
+	if eNode, exists := e.allocator[nodeName]; exists {
 		eNode.isEgressAssignable = isAssignable
 	}
 }
 
-func (oc *Controller) setNodeEgressReady(nodeName string, isReady bool) {
-	oc.eIPC.allocatorMutex.Lock()
-	defer oc.eIPC.allocatorMutex.Unlock()
-	if eNode, exists := oc.eIPC.allocator[nodeName]; exists {
+func (e *egressIPController) setNodeEgressReady(nodeName string, isReady bool) {
+	e.allocatorMutex.Lock()
+	defer e.allocatorMutex.Unlock()
+	if eNode, exists := e.allocator[nodeName]; exists {
 		eNode.isReady = isReady
 	}
 }
 
-func (oc *Controller) setNodeEgressReachable(nodeName string, isReachable bool) {
-	oc.eIPC.allocatorMutex.Lock()
-	defer oc.eIPC.allocatorMutex.Unlock()
-	if eNode, exists := oc.eIPC.allocator[nodeName]; exists {
+func (e *egressIPController) setNodeEgressReachable(nodeName string, isReachable bool) {
+	e.allocatorMutex.Lock()
+	defer e.allocatorMutex.Unlock()
+	if eNode, exists := e.allocator[nodeName]; exists {
 		eNode.isReachable = isReachable
 	}
 }
 
-func (oc *Controller) addEgressNode(egressNode *kapi.Node) error {
+func (e *egressIPController) addEgressNode(egressNode *kapi.Node) error {
 	klog.V(5).Infof("Egress node: %s about to be initialized", egressNode.Name)
-	oc.eIPC.assignmentRetry.Range(func(key, value interface{}) bool {
+	e.assignmentRetry.Range(func(key, value interface{}) bool {
 		eIPName := key.(string)
 		klog.V(5).Infof("Re-assignment for EgressIP: %s attempted by new node: %s", eIPName, egressNode.Name)
-		eIP, err := oc.kube.GetEgressIP(eIPName)
+		eIP, err := e.wf.GetEgressIP(eIPName)
 		if errors.IsNotFound(err) {
 			klog.Errorf("Re-assignment for EgressIP: EgressIP: %s not found in the api-server, err: %v", eIPName, err)
-			oc.eIPC.assignmentRetry.Delete(eIP.Name)
+			e.assignmentRetry.Delete(eIP.Name)
 			return true
 		}
 		if err != nil {
-			klog.Errorf("Re-assignment for EgressIP: unable to retrieve EgressIP: %s from the api-server, err: %v", eIPName, err)
+			klog.Errorf("Re-assignment for EgressIP: unable to retrieve EgressIP: %s from the informer cache, err: %v", eIPName, err)
```

Comment:
Small question: an object not found when calling `GetEgressIP` would mean that the informer looked in its cache, did not find it and then queried the API server for it and did not find it there either, correct? Wouldn't it be more correct to leave the old log message as it was? If that is not the case: then why wasn't L459 changed as well?